.github/worflows/cmake.yml

```diff
@@ -0,0 +1,35 @@
+name: CMake-Manually triggered workflow
+
+on:
+  workflow_dispatch:
```

Comment:
is this valid yml? this might need to be `on: [ 'workflow_dispatch']` (unless you add things under line 4 such as inputs)

Reply:
I took it form this [example](https://docs.github.com/en/actions/reference/events-that-trigger-workflows)

Reply:
looking at that, I think you want `on: workflow_dispatch`

Reply:
Thanks. script updated